frontend/components/site/CommitSummary.jsx

```diff
@@ -49,9 +49,9 @@ function CommitSummary({ buildDetails }) {
         &nbsp;
         <span
           className="commit-time"
-          title={dateAndTime(buildDetails?.createdAt)}
+          title={dateAndTime(new Date(buildDetails?.createdAt))}
         >
-          {timeFrom(buildDetails?.createdAt)}
+          {timeFrom(new Date(buildDetails?.createdAt))}
```

Comment:
This time doesn't represent the commit time (we're duplicating the build time)

Reply:
Not quite following this comment, maybe you could elaborate? is `new Date()` wrong?

Reply:
I declare fixing this is out of scope <3